convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
+The JSON format is meant to contain all details from tests. For human readable
+outputs, use this script to extract the most interest information from your JSON
+reports and write them into an eye-friendly markdown file.
+
+  Typical usage example:
+
+  python convert_results.py --source=results/ --target=result_summaries/
+"""
+
+import argparse
+import json
+import os
+from tabulate import tabulate
+
+
+def summary_table(tests, passed_test_count, total_test_count):
+  """Returns a Markdown table summing passed tests for all tags."""
+  tags = {}
+  for test in tests:
+    pass_score = 1 if test['result'] == 'pass' else 0
+    for tag in test.get('tags', []):
+      if tag in tags:
+        tags[tag][0] += pass_score
+        tags[tag][1] += 1
+      else:
+        tags[tag] = [pass_score, 1]
```

Comment:
why do you need the else? its not obvious -- at least put a comment

Reply:
Obsolete with `defaultdict`.